src/test/java/org/apache/commons/imaging/ImageDumpTest.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.commons.imaging;
+
+import org.junit.Test;
+
+import java.awt.color.ColorSpace;
+import java.awt.image.BufferedImage;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Unit tests for class {@link ImageDump}.
+ *
+ * @date 2017-07-13
+ * @see ImageDump
+ *
+ **/
+public class ImageDumpTest{
+
+
+    @Test
+    public void testDumpColorSpace() {
+
+        ImageDump imageDump = new ImageDump();
+        ColorSpace colorSpace = ColorSpace.getInstance(1004);
+        imageDump.dumpColorSpace("Ku&]N>!4'C#Jzn+", colorSpace);
+
+        assertEquals(3, colorSpace.getNumComponents());
+
+    }
+
+
+    @Test
+    public void testDump() {
+
+        ImageDump imageDump = new ImageDump();
+        BufferedImage bufferedImage = new BufferedImage(10, 10, 10);
+        imageDump.dump(bufferedImage);
+
+        assertEquals(10, bufferedImage.getHeight());
+
+    }
+
+
```

Comment:
Having two blank lines here seems to be inconsistent with the other test classes in commons-imaging. I suggest shortening it to one or zero blank lines.